src/__tests__/integration/unsubscribe.test.js

```diff
@@ -23,7 +29,9 @@ describe('GET /unsubscribe/:token', () => {
   });
```

Comment:
У тесті варто додати перевірку, що підписка видалена з бази даних після виконання запиту.
```suggestion
        const deletedSubscription = await Subscription.findOne({ where: { email: testEmail } });
        expect(deletedSubscription).toBeNull();
  });
```